packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md

```diff
@@ -4,139 +4,105 @@ excerpt: >-
 
-Without the client-based rate limit framework, Bob consumes all of the total allowed requests per minute (2,000 requests per minute). This results in HTTP 429 errors for both Bob and Alice, which makes the application inaccessible for everyone.
+* You've hit a rate limit warning or violation, and a rate limit event (violation or warning) is created in the System Log from where you can click **View Utilization** under **Event info** to go to the rate limits dashboard. All the rate limit events in the System Log (violations and warnings) have a link to the rate limit dashboard.
 
-With the client-based rate limit framework enabled, after Bob exceeds his individual limit of 60 requests per minute, HTTP 429 errors are sent. Requests that originate from Bob's IP address, and the specific device from which the requests are made, start receiving the errors. Alice continues to access the application without any issues.
+![Access the dashboard by link in the System Log](/img/RLDashboard3.png "Displays the event in the System Log that notifies you of a rate limit violation with the link to the endpoint's rate limit dashboard")
 
-![Client-based isolation for a unique IP address](/img/diagram1.png "Displays Bob as portal123 IP1 and Alice as portal123 IP2, both making authorize requests")
+## Key components of the dashboard
 
-#### Client-based isolation for users accessing `/authorize` from a NAT IP
+The rate limits dashboard provides utilization for the endpoint based on a set of statistics and shows multiple views of the data utilization.
 
-Alice, Bob, and Lisa all work from the same office. Since they access Okta through a Network Address Translation (NAT) IP from their office network, everyone shares the same IP address. When they make the authorize request to `https://company.okta.com/oauth2/v1/default/authorize?clientId=portal123`, the client-based rate limit framework creates a unique per minute request quota from the combination of every user's IP address, OAuth client ID of the application, and the device identifier set by Okta in each user's browser.
+![Summary of the key components of the dashboard](/img/RLDashboard4.png "Displays the endpoint's utilization and the multiple views of the data utilization")
 
-![Client-based isolation for users accessing the authorize endpoint from a NAT IP](/img/diagram2.png "Displays Alice as portal123 Device1, Bob as portal123 Device2, Lisa as portal123 Device3, all making authorize requests that are a combination of the shared IP address")
+The dashboard helps you to investigate the rate limit or violation issue and to prevent subsequent rate limit issues from occurring. To view the most recent data, refresh your screen. There may be a slight delay of two to four minutes in obtaining new data, so you should wait a moment before you refresh your screen.
```

Comment:
```suggestion
The dashboard helps you to investigate the rate limit or violation issue and to prevent subsequent rate limit issues from occurring. To view the most recent data, refresh your screen. There may be a slight delay of two to four minutes in obtaining new data.
```

I don't think that last piece is necessary to convey your meaning. LMK if you disagree!